### ISSUE TITLE
Constraint violation when you create transaction

### ISSUE THREAD

**Issue by rained23** [depsimon/laravel-wallet#4]
Hi,

Just playing out with the package, if you try to create transaction on fresh user, it will throw an exception.

**Comment by depsimon** [comment]
What is the exception?

**Comment by rained23** [comment]
```
local.ERROR: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'wallet_id' cannot be null
```

in your deposit/withdraw function have $this->wallet->balance that create the wallet object but if you use the failDeposit it will fail because it try to create transaction without wallet_id.

**Comment by depsimon** [comment]
Thanks. I've just published [v1.0.8](https://github.com/depsimon/laravel-wallet/releases/tag/v1.0.8) with the fix. Make sure to add a default value to the balance in the wallets table.
